app/templates/preprints/provider/preprint-detail.hbs

```diff
@@ -20,7 +20,7 @@
     {{#if fetchData.isRunning}}
         <div class="preprint-status-component preprint-status-component--skeleton"></div>
     {{else}}
-        {{preprint-status-banner submission=preprint saving=savingAction submitDecision=(action 'submitDecision')}}
+        {{preprint-status-banner submission=preprint saving=savingAction setUserEnteredReview=(action 'setUserEnteredReview') submitDecision=(action 'submitDecision')}}
```

Comment:
You could use the [`mut` helper](https://emberjs.com/api/ember/2.16/classes/Ember.Templates.helpers) here, like `setUserEnteredReview=(action (mut userHasEnteredReview))`, which would let you delete `setUserEnteredReview`.